Resources/Prototypes/_LostParadise/Entities/Clothing/Head/hardsuit-helmet.yml

```
@@ -862,3 +862,84 @@
         Slash: 0.9
         Piercing: 0.9
         Radiation: 0.9
+
+- type: entity
+  parent: ClothingHeadHardsuitWithLightBase
+  id: LPPClothingHeadHelmetHardsuitAcientHos
+  categories: [ HideSpawnMenu ]
+  name: NT-014 helmet
+  description: Good helmet for every needs.
+  components:
+  - type: BreathMask
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)